docs/sources/administration/cli.md

```diff
@@ -1,6 +1,7 @@
 ---
 aliases:
-  - administration/cli/
+  - cli/ # docs/grafana/latest/adminsitration/cli
+  - ../cli/ # docs/grafana/latest/cli
```

Comment:
Here's something fun, we've now moved this page back to where it was originally so we no longer need the alias :) 

```suggestion
  - ../cli/ # /docs/grafana/latest/cli/
```